src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,25 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        if (email != null && password != null && email.length() > 0 && password.length() > 0) {
+
+            UserService us = new UserService();
```

Comment:
```suggestion
            UserService userService = new UserService();
```
don't use shortened names - they are not informative